sdk/core/pom.xml

```diff
@@ -97,16 +97,19 @@
             <groupId>org.jacoco</groupId>
             <artifactId>jacoco-maven-plugin</artifactId>
             <version>0.8.5</version> <!-- {x-version-update;org.jacoco:jacoco-maven-plugin;external_dependency} -->
+            <configuration>
+              <outputDirectory>${project.reporting.outputDirectory}/test-coverage</outputDirectory>
+              <excludes>
+                <exclude>META-INF/**</exclude>
+              </excludes>
+            </configuration>
             <executions>
               <execution>
                 <id>report-aggregate</id>
                 <phase>verify</phase>
                 <goals>
                   <goal>report-aggregate</goal>
                 </goals>
-                <configuration>
-                  <outputDirectory>${project.reporting.outputDirectory}/test-coverage</outputDirectory>
```

Comment:
>outputDirectory [](http://example.com/codeflow?start=55&length=15)

should this not be added to the parent directly? So not all SDKs have to add it.

Reply:
This is the root-level service POM which doesn't use the SDK parent POM

Reply:
Ahh, missed that, thanks!